src/gatsby/src/components/advisories/advisoryFilter.js

```diff
@@ -19,12 +25,35 @@ const AdvisoryFilter = ({
   const [isKeywordFilter, setIsKeywordsFilter] = useState(getFilter("keyword"))
   const [eventText, setEventText] = useState("")
   const [selectedEventType, setSelectedEventType] = useState([defaultEventType])
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false)
+  const typeaheadRef = useRef(null)
+  const hasDefaultEventType = selectedEventType[0]?.label === "All"
 
-  // Local handlers, calls to parent methods
-  // will trigger useEffect functions in parent
+  // functions
+  const hasResult = (text) => {
+    const eventTextLower = text.toLowerCase()
+    const results = eventTypes.filter(type =>
+      type.label.toLowerCase().startsWith(eventTextLower) || type.label.toLowerCase().includes(eventTextLower)
```

Comment:
is the `startsWith` part needed here? Won't it be covered by the `includes` check?

Reply:
Good catch!